### PR TITLE
add notebooknodeexplorer shortcut properties

### PR DESCRIPTION
**Pull request by feloxx**
Add notebooknodeexplorer global shortcut

In `NotebookNodeExplorer::setupShortcuts` impl

- properties

### CODE REVIEW

File: src/widgets/notebooknodeexplorer.cpp [hunk 1]
```diff
@@ -2251,6 +2246,25 @@ void NotebookNodeExplorer::openSelectedNodesWithProgram(const QString &p_name, b
     }
 }
 
+void NotebookNodeExplorer::openSelectedNodesProperties(bool p_master)
```
Comment (tamlok) [inline src/widgets/notebooknodeexplorer.cpp:2249]: `openCurrentNodeProperties()`?

File: src/widgets/notebooknodeexplorer.cpp [hunk 2]
```diff
@@ -2251,6 +2246,25 @@ void NotebookNodeExplorer::openSelectedNodesWithProgram(const QString &p_name, b
...
+{
+    auto node = p_master ? getCurrentMasterNode() : getCurrentSlaveNode();
```
Comment (tamlok) [inline src/widgets/notebooknodeexplorer.cpp:2251]: Check selected nodes count first here.

**Review by tamlok — CHANGES_REQUESTED** [review]

File: src/widgets/notebooknodeexplorer.cpp (hunk 1 above)
Comment (feloxx) [reply src/widgets/notebooknodeexplorer.cpp:2249]: done

File: src/widgets/notebooknodeexplorer.cpp (hunk 2 above)
Comment (feloxx) [reply src/widgets/notebooknodeexplorer.cpp:2255]: done

**Pull request merged**